src/main/java/seedu/address/model/person/PersonIsInModulePredicate.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+import java.util.function.Predicate;
+
+import seedu.address.model.module.Module;
+
```

Comment:
```suggestion
@Test
public void test_withPersonInSetOfPersons_returnsTrue() {
}

@Test
public void test_withoutPersonInSetOfPersons_returnsFalse() {
}

@Test
public void test_withNull_returnFalse() { // I am not sure if this will have error or be False
}

@Test
public void equals() {
    // Same object
    // With null
    // With different Predicate type
    // With different Person set
}
```
Might it be possible to include some test cases for this?

Reply:
Yes, I was waiting for the `add-person-to-module` command to be implemented, but thanks for the foundation provided here!